static/js/dev.ts

```diff
@@ -239,6 +261,48 @@ window.onload = () => {
       ]),
     ];
   }
+
   containerId = addChartContainer(1000, 500);
-  drawGroupLineChart(containerId, 1000, 500, dataGroupsDict);
+  drawGroupLineChart(
+    containerId,
+    1000,
+    500,
+    dataGroupsDict,
+    computePlotParams(dataGroupsDict)
+  );
+
+  containerId = addChartContainer(1000, 500);
+  let drawChartParams: ChartRegionPropsType;
+  drawChartParams = {
+    chartElem: containerId,
+    placeIds: ["geoId/05"],
+    statVarsAndMeasuredProps: [
+      ["Count_Person", "count"],
+      ["Count_Person_Male", "count"],
+      ["Median_Age_Person", "age"],
+    ],
+    perCapita: false,
+  };
+
+  ReactDOM.render(
+    React.createElement(ChartRegion, drawChartParams),
+    document.getElementById(containerId)
+  );
+
+  containerId = addChartContainer(1000, 500);
+  drawChartParams = {
+    chartElem: containerId,
+    placeIds: ["geoId/05", "geoId/06"],
+    statVarsAndMeasuredProps: [
+      ["Count_Person", "count"],
+      ["Count_Person_Male", "count"],
+      ["Median_Age_Person", "age"],
+    ],
+    perCapita: false,
+  };
+
+  ReactDOM.render(
```

Comment:
can move these to dev_menu.ts

Reply:
Done, the things shown in dev_menu page is only for test, so the UI is currently not considered.